src/accordion.stories.ts

```diff
@@ -44,7 +44,7 @@ const meta: Meta = {
       });
   },
   render(arguments_) {
-    /* eslint-disable @typescript-eslint/no-unsafe-argument */
+    /* eslint-disable @typescript-eslint/no-unsafe-argument, @typescript-eslint/prefer-nullish-coalescing */
```

Comment:
A useful rule overall, I think, but it has its share of false positives.

Reply:
Rather than having to disable it in so many separate places, should we add an override, disabling it for all `*.stories.*` files?